harness/determined/searcher/_search_method.py

```diff
@@ -229,9 +299,43 @@ def on_validation_completed(
         self, searcher_state: SearcherState, request_id: uuid.UUID, metric: Any, train_length: int
     ) -> List[Operation]:
         """
-        Informs the searcher that the validation workload
-        initiated by the same searcher has completed after training for ``train_length`` units.
-        It returns any new operations as a result of this workload completing.
+        Informs the searcher that the validation workload has completed after training for
+        ``train_length`` units. It returns any new operations as a result of this workload
+        completing
+
+        Example:
+
+        .. code:: python
+
+            def on_validation_completed(
+                self,
+                searcher_state: SearcherState,
+                request_id: uuid.UUID,
+                metric: Any,
+                train_length: int
+            ) -> List[Operation]:
+                MAX_TRAIN_LENGTH = 10  # Best defined in __init__
+                if not searcher_state.experiment_completed \\
```

Comment:
I don't think it's possible to reach this if-statment if either of `searcher_state.experiment_completed` or `searcher_state.experiment_failed` are `True`, right?